honestcash-v2/src/app/auth/models/authentication.ts

```diff
@@ -31,7 +31,7 @@ export interface LoginContext {
 
 export interface LoginSuccessResponse {
   user: User;
-  wallet: Wallet;
+  wallet: ISimpleWallet;
```

Comment:
wallet may also be undefined or null. in that case walletService is clever enough to return a wallet always (if simple-bitcoin-wallet is loaded of course)